templates/windows/chocolateyInstall.ps1.erb

```diff
@@ -0,0 +1,28 @@
+$toolsDir = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+$fileList = Join-Path -path $toolsDir -childpath "file-list.txt"
+
+if (!(Test-Path -path "$fileList")) {
+  Throw "Unable to find file '$fileList', cannot proceed with install"
+}
+
+$lines = Get-Content "$fileList"
+foreach ($destination in $lines) {
+  $originFile = "$destination" -replace '^[a-zA-Z]:/',''
+  $origin = Join-Path -path "$toolsDir" -childpath "$originFile"
+  if (Test-Path -path "$origin") {
+    $parent = Split-Path -path "$destination" -parent
+    if (!(Test-Path -path "$parent")) {
+      New-Item -ItemType directory -Path "$parent"
+    }
+    if (Test-Path -path "$destination") {
+      Write-Debug "Overwriting '$destination'"
+    }
+    Copy-Item -path "$origin" -destination "$destination" -Force
+  } else {
+    # If the item we are trying to copy over does not exist in our source directory,
+    # we assume it is an empty directory and simply create one in its place. There is
+    # a possibility that this will hide an error where there is actually a missing
+    # file. However, this is such a slim possibity, this action was deemed safe.
+    New-Item -ItemType directory -Path "$destination"
+  }
+}
```

Comment:
It was originally, but then I changed it and forgot to make it not an erb...  good point. I should probably change that